ComputeClient/Compute.Client.UnitTests/SampleOutputs/ListSnapshotServicePlansResponse.xml

```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<snapshotServicePlans xmlns="urn:didata.com:api:cloud:types" pageNumber="1" pageCount="1" totalCount="5" pageSize="5" >
+	<snapshotServicePlan id="ONE_MONTH">
```

Comment:
Do we need a test case where supportsReplication is true?

Reply:
good shout will add one for completeness